nodes/modifier_make/contour2D.py

```diff
@@ -122,14 +125,14 @@ def mask_by_distance(verts, parameters, modulo, edges, mask_t):
     net = parameters[3]
     mask = []
 
-    sides_space = sides_space_limits(or_verts, or_radius, net, edges, mask_t, modulo)
+    sides_space, sides_space_bbox = sides_space_limits(or_verts, or_radius, net, edges, mask_t, modulo)
     poli_ang_list = [pi/v for v in or_vert_num]
-
+    poli_ang_cos_list = [cos(ang) for ang in poli_ang_list]
```

Comment:
np.cos(poli_ang_list).tolist()
?
or maybe even without tolist()
Need to test if it will be faster, off course.